server/models/server-service.js

```diff
@@ -0,0 +1,113 @@
+var async = require('async');
+var debug = require('debug')('MeshModels.server.ServerService');
+var fs = require('fs');
+var util = require('util');
+
+module.exports = function(ServerService) {
+  ServerService.disableRemoteMethod('upsert', true);
+  ServerService.disableRemoteMethod('updateAll', true);
+
+  ServerService.observe('before save', function(ctx, next) {
+    var Group = ServerService.app.models.Group;
+
+    if (ctx.instance) {
+      // Full save of Service
+      var service = ctx.instance;
+      if (service._groups.length === 0) {
+        service._groups.push(new Group({id: 1, name: 'default', scale: 1}));
+      }
+      if (service._groups.length === 1 && service._groups[0].scale <= 0) {
+        service._groups[0].scale = 1;
+      }
+      process.nextTick(next);
+    }
```

Comment:
This is only setting default groups for a new service that is created without groups. AFAIK the multi service case is only hit when updating multiple models, not creating new ones.